Decred Wallet/Custom Views/FloatingLabelTextField.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  FloatingLabelTextField.swift
+//  Decred Wallet
+//
+//  Created by Sprinthub on 15/08/2019.
+//  Copyright © 2019 Decred. All rights reserved.
+//
+
+import UIKit
+
+class FloatingLabelTextField: UITextField {
+    
```

Comment:
These 2 empty line spaces have no use.

Reply:
resolved in e7f7df7

Reply:
This isn't resolved, I still see 2 empty line spaces.

Reply:
this file was later renamed to `FloatingLabelTextInput` disregard